stylesheets/_conversation.scss

```diff
@@ -927,6 +927,33 @@ span.status {
 
   form.send {
     background: #ffffff;
+
+    &.video-attachment {
+      .image-container {
+        position: relative;
+      }
+
+      .outer {
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+
+        display: flex;
+        text-align: center;
+        display: flex;
```

Comment:
Duplicate property? I’ve run into this before myself so I at times used 🔤order for CSS declarations but got some dirty looks from co-workers 😉 Maybe we can impose a fixed order via a linter, preferably using auto-format to avoid such issues 😄